doc/orchestrator.rst

```diff
@@ -165,3 +165,511 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrator Example
+=============================
+SmartSim offers functionality to automate the deployment of multiple
+databases, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+In the following sections, we simulate a simple version of this use case.
+
+The example is comprised of two script files:
+
+* The :ref:`Application Script<The Application Script>`
+* The :ref:`Experiment Script<The Experiment Script>`
+
+**The Application Script demonstrates:**
+
+1. Connecting SmartRedis clients within the application to retrieve tensors
+   from the standard databases to store in a colocated database.
+
+**The Experiment Script demonstrates:**
+
+1. Launching two standard Orchestrators with unique identifiers.
+2. Creating a colocated model (launches one colocated Orchestrator) with
+   a unique identifier that inherits the application script.
+3. Connecting SmartRedis clients within the driver script to send tensors to standard Orchestrators
+   for retrieval within the application.
+
+The Application Script
+----------------------
+To store and retrieve data to and from databases
```

Comment:
Reword the opening sentence to:
"Applications interact with the databases through a SmartRedis client".